### DISCUSSION TITLE
Crear repositorio de Cuantica

### DISCUSSION THREAD

**Discussion by gastonstrizzolo** [FAMAF-resources/Welcome#12] [Ideas]
Buenas! Se podría crear un repositorio para la materia de cuántica de Lic en Computacion? El nombre completo de la materia es "Procesamiento Cuántico de la Información"

**Comment by shirosweets** [comment]
Hola, si, ahora lo creamos!

**Comment by shirosweets** [comment]
https://github.com/FAMAF-resources/Optativa-Procesamiento_Cuantico_de_la_Informacion-FAMAF
